flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -239,6 +248,7 @@ class DataStreamOverAggregate(
   }
 
   def createBoundedAndCurrentRowOverWindow(
```

Comment:
I do not think we need add this check. Because both `row-base` and `time-base` will be able to meet the incorrect result due to state cleanup. an importation thing we need to do is let use know the function of the `cleanup config`.